contrib/subprojects/tinyendian.wrap

```diff
@@ -0,0 +1,5 @@
+[wrap-git]
+directory = tinyendian
+
+url=https://github.com/ximion/tinyendian
```

Comment:
I have [opened a request](https://github.com/kiith-sa/tinyendian/issues/4) to bring tinyendian to the org, so you would be able to upstream your meson work ;)